dist/index.html

```diff
@@ -1,20 +1,111 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <title>My JavaScript App</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-</head>
-<body>
-  <header>
-
-  </header>
-  <main>
-
-  </main>
-  <footer>
-
-  </footer>
-  <script src="app.bundle.js" type="text/javascript"></script>
-</body>
+
+    <head>
+      <meta charset="utf-8">
+      <title>Trekkin'</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+
+    <body>
+
+      <header>
+        <h1>Trekkin' - Go Global!</h1>
+
+        <section id="status-messages">
+          <ul>
+          </ul>
+        </section>
+      </header>
+
+      <aside class="columns large-6 small-12">
+        <section id="trip-description"></section>
+
+        <h3>Reserve a Trip!</h3>
+
+        <form id="reservation-trip-form" method="post">
+          <label for="name">Full Name</label>
+          <input type="text" name="name"></input>
```

Comment:
This isn't being selected by `app.js`  Because it's having trouble with `$('input[name="name"]')`  Instead an alternative way of being selected would be appropriate, an id etc.